experimental/Schemes/src/Resolution_structure.jl

```diff
@@ -364,10 +369,93 @@ function add_map_embedded!(f::BlowUpSequence, phi::BlowupMorphism)
   return f
 end
 
+function extend(f::MixedBlowUpSequence, g::Union{BlowUpSequence,MixedBlowUpSequence})
```

Comment:
```suggestion
function extend!(f::MixedBlowUpSequence, g::Union{BlowUpSequence,MixedBlowUpSequence})
```
since this seems to mutate `f`

Reply:
done